components/03-modules/dashboard-nav/_dashboard-nav.scss

```diff
@@ -0,0 +1,50 @@
+$dashboard-nav-padding: 15px !default;
+$dashboard-nav-margin : 15px !default;
+
+.dashboard-nav {
+    display: flex;
+    flex-direction: column;
+    background-color: $gray-lighter;
+    padding: $dashboard-nav-padding;
+    margin-bottom: $dashboard-nav-margin;
+
+    &__content {
+        padding: $dashboard-nav-padding 0;
+        border-top: 1px solid $gray;
+    }
+
+    &__item {
+        padding: 15px;
+        transition: $transition-base;
+
+        &:hover {
+            background-color: $white;
+        }
+
+        &:first-child {
+            padding-top: 15px;
+        }
+
+        &:last-child {
+            padding-bottom: 15px;
+        }
+    }
+
+    &__title {
+        margin-bottom: $dashboard-nav-margin;
+        padding-bottom: 10px;
+        border-bottom: 1px solid $gray;
+    }
+
+    .current {
+        border-left: 2px solid $red;
+        color: $red;
+        padding-left: 23px;
+        position: relative;
+        margin-left: -15px;
+
+        &:hover {
+            background-color: inherit;
+        }
+    }
+}
```

Comment:
Dashboard nav should be toggled (dropdown) on mobile view, b/c it takes too much place when it's above the content in smallest views (tablet, mobile). Please correct

Reply:
Added 👍

Reply:
👍 
too much space below closed navigation:
<img width="631" alt="screen shot 2017-09-07 at 11 25 16" src="https://user-images.githubusercontent.com/15080214/30156184-47bdc030-93bf-11e7-8e93-a5a377900ed9.png">

Reply:
Removed

Reply:
👍